### DISCUSSION TITLE
[suggestion]Categorize by folder

### DISCUSSION THREAD

**Discussion by Xraytest** [navidrome/navidrome#3077] [Ideas]
add classification that about folder
my version:[0.47.0] (8c7d95c1)
it may have already been done in newer version

**Comment by deluan** [comment]
Thanks for using Navidrome. We have no plans to implement this feature. See https://www.navidrome.org/docs/faq/#-can-you-add-a-browsing-by-folder-optionmode-to-navidrome.

**Comment by Glandos** [comment]
Duplicate of https://github.com/navidrome/navidrome/discussions/2414
